vendor/github.com/golang-migrate/migrate/v4/database/multistmt/parse.go

```diff
@@ -0,0 +1,46 @@
+// Package multistmt provides methods for parsing multi-statement database migrations
```

Comment:
Why the `vendor` directory is on the top-level? As for now, it's specific to a single module, so I think it should reside within the `data-store`.

Reply:
That depends if other services will use those dependencies ie. are written in go. If so it should be in parent directory of both folders
* zpi/
* * frontend/
* * vendor/
* * data-store/
* * clap/ 

i can move it to 

* zpi/
* * frontend/
* * backend/
* * * vendor/
* * * data-store/
* * * clap/ 

if other services are not written in go, then yes. we can move it to `data-store`. I'd leave it there until we know what other services are written in

Reply:
Alright, let's keep it in mind.